app/assets/stylesheets/app/global.scss

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,400i,700,800');
+
+html {
+  *, *:before, *:after {
+    box-sizing: border-box;
+  }
+}
+
+html,
+body {
+  min-height: 100%;
+}
+
+body {
+  font-family: $font-sans;
+  font-size: 18px;
+  background: $backgroundColor;
+  background-image: linear-gradient(to bottom, $backgroundColor, #D94A4A);
+  color: white;
+}
+
+a {
+  color: white;
+}
+
+.app-page {
+  padding-top: 80px;
+
+  @include for-tablet-portrait-up {
+    padding-top: 120px;
```

Comment:
💅 wdyt of leaving always as 80px? I feel that 120px is too much, kinda not needed as 80px is already a huge space to begin showing the content. Its not even like there is another section above and we wanna make sure the user understands that they are 2 different sections/topics.

wdyt?

Reply:
Sure man